components/tidb_query_codegen/Cargo.toml

```diff
@@ -8,8 +8,8 @@ publish = false
 proc-macro = true
 
 [dependencies]
-darling = "0.9"
-syn = "0.15"
-quote = "0.6"
-proc-macro2 = "0.4"
+darling = "0.10"
+syn = { version = "1", features = ["full", "extra-traits"] }
```

Comment:
Hey, cool! Do you think you could update it in `components/match_template` too? This would keep the overall build graph of TiKV simpler. :)

Reply:
That's a good point! I will open an issue to the community after this PR.